bzt/modules/reporting.py

```diff
@@ -175,6 +180,103 @@ def __report_failed_labels(self, cumulative):
                 if failed_samples_count:
                     self.log.info(report_template, failed_samples_count, sample_label)
 
+    @staticmethod
+    def __set_header(table_headers, table_headers_desc):
+        table_header = []
+        header_index = 0
+        justify_columns = {}
+        for header in table_headers:
+            table_header.append(table_headers_desc[header].split(":")[0])
+            justify_columns[header_index] = table_headers_desc[header].split(":")[1]
+            header_index += 1
+        return table_header, justify_columns
+
+    @staticmethod
+    def __set_table_data(data, table_headers):
+        table_data = []
+        for element in data:
+            table_item = []
+            for header in table_headers:
+                table_item.append(element[header])
+            table_data.append(table_item)
+        return table_data
+
+    def __get_table(self, header, data, title=""):
+        table_headers = header["headers"]
+        table_headers_desc = header["descriptions"]
+        table_data = []
+        table_header, justify_columns = self.__set_header(table_headers, table_headers_desc)
+        table_data.append(table_header)
+        table_data.extend(self.__set_table_data(data, table_headers))
+        table_instance = AsciiTable(table_data, title)
+
+        table_instance.justify_columns = justify_columns
+
+        return table_instance.table.splitlines()
+
+    def __console_safe_encode(self, text):
+        return text.encode(locale.getpreferredencoding(), errors='replace').decode('unicode_escape')
+
+    def __get_scenario_label_name(self, sample_label):
```

Comment:
This piece is somethign custom, let's remove "scenario" column for now, until we have better solution